plasma_cash/child_chain/event.py

```diff
@@ -0,0 +1,13 @@
+callback = {}
+
+
+def on(event):
```

Comment:
as everything is relying on this, and probably not too many future changes going on here (I guess?), I think this file should be covered by test. Since this file is so simple, probably we can just decorate it on a test function and test by emit? ( i will leave this to you

Reply:
Agree.